### ISSUE TITLE
Pathing issue

### ISSUE THREAD

**Issue by Spiritwo** [CitizensDev/Citizens#761]
I believe that the max distance between two points has a glitch.  After placing a few points, it doesn't let me place any more, stating that "Points can't be more than 16 blocks away from each other".  I get this message even if the next point in trying to place is right next to the previous point.  I believe that it has started to read the distance as a measurement from the first point, and only the first point, not each consecutive point.  There are no errors in the console when this message comes up.

**Comment by charlie1990** [comment]
it isn't a glitch, just bad explaining..if you want to change the max distance on waypoints go into the citizens.yml and changed pathfinding to a higher number

**Comment by JaredTS486** [comment]
Yes i changed mine to 100. It complains that the final point is our of range of the initial. My NPC's are making big loops around my town.

**Comment by charlie1990** [comment]
Hmm never had any problems with it, I have my pathfinding set to 175. One thing I can say is that npc's that are being made to walk far(80+ is a good number) should have a decent amount of waypoints, don't space them out really far(keep em to 1 waypoint every 15 blocks i'd say) it really helps in stopping them from making very many mistakes and missing waypoints.
